LayoutTests/fast/css/getComputedStyle/getComputedStyle-webkit-mask-box-image.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<script src="../../../resources/js-test-pre.js"></script>
```

Comment:
nit: js-test-pre.js / js-test-post.js should not be used in new code. Please use `js-test.js` or better yet the WPT testharness.js file.

Reply:
Changed it to use testharness.js